solutions/swb-reference/README.md

```diff
@@ -68,11 +68,232 @@ This step is necessary to setup Service Catalog portfolio and products
+
+    3. In `./integration-tests/config` directory assign `id` value to `envTypeId` property and `type` value to `envType` property in `<STAGE>.yaml` file 
+
+
+3. For `envTypeConfigId` open Postman Collection and select `List envTypeConfigs` inside `envTypeConfig` folder.
+    
+    1. Replace `:envTypeId` in the URL request `{{API_URL}}/environmentTypes/:envTypeId/configurations` with value of the environment type id from the previous step.
+
+    2. Excecute `List envTypeConfigs` request, you should get a json response with the next information
+        ```
+        {
+            "data": [
+                {
+                    "createdAt": "2022-08-11T15:29:15.935Z",
+                    "updatedBy": "########-####-####-####-############",
+                    "createdBy": "########-####-####-####-############",
+                    "name": "Config 1",
+                    "allowedRoleIds": [],
+                    "resourceType": "envTypeConfig",
+                    "provisioningArtifactId": "<artifact id>",
+                    "params": [
+                        {
+                            "value": "${iamPolicyDocument}",
+                            "key": "IamPolicyDocument"
+                        },
+                        {
+                            "value": "ml.t3.medium",
+                            "key": "InstanceType"
+                        },
+                        {
+                            "value": "0",
+                            "key": "AutoStopIdleTimeInMinutes"
+                        },
+                        {
+                            "value": "0.0.0.0/0",
+                            "key": "CIDR"
+                        }
+                    ],
+                    "updatedAt": "2022-08-11T15:29:15.935Z",
+                    "sk": "ET#<id number>}",
+                    "owner": "########-####-####-####-############",
+                    "description": "Description for config 1",
+                    "id": "########-####-####-####-############",
+                    "pk": "ETC",
+                    "productId": "<product id>",
+                    "type": "sagemakerNotebook"
+                }
+            ]
+        }
+        ```
+        If there are no environment type configs displayed please follow instructions [here](../swb-reference/SETUP_v2p1.md###setup-project-configurations,-environmentType,-and-environmenttypeconfig) to create a new environment type config.
+
+    3. Choose the Environment Type Config that integration test will use as default when creating any Environment and copy the `id` value from the request.
+
+    4. In `./integration-tests/config` directory assign value copied to `envTypeConfigId` property in `<STAGE>.yaml` file 
+
+
+4. For `projectId` open Postman Collection and select `List projects` inside `projects` folder.
```

Comment:
I believe for SWB V1, we recommended creating a separate project specifically for integ tests. Should we do that as well here? It might be confusing to have integ test resources within a legit project. Obviously, this is more for our developing customers so it might not be as big of a concern.

Reply:
good point I will add that recommendation to the readme

Reply:
readme updated now

Reply:
This recommendation will be implemented in the future since there is no endpoint to create a project currently.